SetupInstall-Centos7.sh

```diff
@@ -4,6 +4,22 @@
 source etc/common.sh
 
 
+# ----- Install the required gpu software on the host ----- #
+install_gpu_software()
+{
+  echo "Installing nvidia-docker2..."
+  # Add the package repositories
+  distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+  curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.repo | \
+  sudo tee /etc/yum.repos.d/nvidia-docker.repo
```

Comment:
Do we really need to add the repo file?
Cannot we just install the package without messing around with the local manager config?

Reply:
Fixed.